src/bot/modules/reminders.ts

```diff
@@ -118,9 +119,11 @@ export default class Reminders {
           .map((members_id) => godfathersEmojis.find(({ id }) => members_id === id)?.emoji)
           .filter((e) => e)
           .join(' ');
-        const line = `[${proposal.type.toLowerCase()}](https://discord.com/channels/${guild.id}/${
-          proposal.type === ProposalType.SUGGESTION ? suggestionsChannelId : correctionsChannelId
-        }/${proposal.message_id}) ${godfathers}\n`;
+        const line = `[Voir la ${proposal.type.toLowerCase()}](${messageLink(
```

Comment:
Je serais plus pour un design du style :
```suggestion
        const line = `[[${proposal.type === ProposalType.SUGGESTION ? 'Suggestion' : 'Correction'}]](${messageLink(
```